spec/factories/wishlists.rb

```diff
@@ -2,11 +2,13 @@
   factory :wishlist do
     name "Birthday"
     description "Tom's 18th birthday"
+    user
   end
 
   factory :invalid_wishlist, class: Wishlist do
     name ""
     description ""
     user_id nil
+    user
```

Comment:
Invalid is invalid. Shouldn't have a user? Either way, you should set `user_id` or use the association helper `user`, not both.